react/finger-exercises/redux/02 - react-redux/src/app/screens/App/components/ShoppingCart/components/Item/index.js

```diff
@@ -1,32 +1,28 @@
-import React, { PureComponent } from 'react';
+import React, { Component } from 'react';
 import { func } from 'prop-types';
 import { bookSelectedPropType } from '@constants/propTypes';
 import Button from '@components/Button';
+import { connect } from 'react-redux';
+import cartActionsCreators from '@redux/cart/actions';
 
 import styles from './styles.scss';
 
-class Item extends PureComponent {
-  addItem = () => {
-    const { item, addItem } = this.props;
-    addItem(item.id);
-  };
-
-  removeItem = () => {
-    const { item, removeItem } = this.props;
-    removeItem(item.id);
-  };
-
+class Item extends Component {
+  handlePlusClick = () => this.props.addItem(this.props.item.id);
+  handleTrashClick = () => this.props.removeItem(this.props.item.id);
```

Comment:
Add a space after and before of this function

Reply:
linter don't let me do this, it appear as an error

Reply:
Already fixed this, i don't know why it doesn't appear outdated.